lbz/events/enums.py

```diff
@@ -0,0 +1,17 @@
+class CognitoEventType:
+    # docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-custom-email-sender.html
```

Comment:
It needs to be a full URL, otherwise, e.g. PyCharm does not recognize it:
```suggestion
    # https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-custom-email-sender.html
```